drivers/unix/net_socket_posix.cpp

```diff
@@ -81,6 +82,7 @@
 #define SOCK_BUF(x) (char *)(x)
 #define SOCK_CBUF(x) (const char *)(x)
 #define SOCK_IOCTL ioctlsocket
+#define SOCK_FIONREAD_LEN_TYPE unsingned long
```

Comment:
This won't compile :)

Reply:
🤦🏼‍♂️ I think I should try to fix bugs at noon and not near midnight.